### ISSUE TITLE
Setting up the bot

### ISSUE THREAD

**Issue by Tookie111** [chiyadev/MudaeFarm#123]
just some questions i have that have been confusing me recently
the first is that when "state_update_command" is "" then the bot either just doesnt roll or roll every 5-10 minutes, but ive heard that if i set it to "$tu" then it wont claim any characters from my wishlist. How should i set it up so that it rolls faster but not when i cant claim or roll anymore but also be able to claim characters from my wishlist.

also, if you set "interval_override_minutes" to 4 for example then will it keep rolling every 4 minutes regardless of whether i can claim/actually roll or not? otherwise i will just set that up but it would be a problem if it keeps rolling even if i cant claim

thanks in advance

**Comment by Tookie111** [comment]
sorry that github sent 2 issues instead of just the 1 btw even tho i only posted it once

**Comment by phosphene47** [comment]
`state_update_command` is used by the bot to determine the amount of rolls you have left and when it will be reset, by sending `$tu` periodically. Setting it to `""` will disable it, causing the bot will attempt to claim everything in the wishlist regardless of cooldown.

> but ive heard that if i set it to "$tu" then it wont claim any characters from my wishlist. How should i set it up so that

That is not the expected behavior, so it's probably a bug. I haven't updated this bot in months, so it may be broken.

> also, if you set "interval_override_minutes" to 4 for example then will it keep rolling every 4 minutes regardless of whether i can claim/actually roll or not? otherwise i will just set that up but it would be a problem if it keeps rolling even if i cant claim

`interval_override_minutes` simply overrides the adaptive rolling interval. Adaptive rolling interval is determined by the number of rolls you have left, the time remaining until your rolls are refilled, and whether or not you are on cooldown. `state_update_command` is used to retrieve these data.

In other words, once you disable `state_update_command`, the bot will attempt to claim everything you wish. It will roll as frequently as specified by `interval_override_minutes`, even if you can't claim. It knows nothing about cooldowns.

You can still set `interval_override_minutes` without disabling `state_update_command`.

**Comment by phosphene47** [comment]
[v4](https://github.com/chiyadev/MudaeFarm/releases/tag/v4.0.0) no longer uses `state_update_command`.
